txnkv/transaction/2pc.go

```diff
@@ -1014,7 +1011,12 @@ func (c *twoPhaseCommitter) doActionOnGroupMutations(bo *retry.Backoffer, action
 					zap.Error(e))
 				metrics.SecondaryLockCleanupFailureCounterCommit.Inc()
 			}
-		})
+		}
+		if gP, ok := c.store.(interface{ Go(func()) }); ok {
+			gP.Go(f)
+		} else {
+			go f()
+		}
```

Comment:
I can imagine the goroutine pool could be used elsewhere. Then, this snippet may be repeated.

So, do you think it acceptable to make `Go(func())` a function inside of `kvstore`? Indeed, this seems a bit strange considering the interface name is `kvstore`. But `kvstore` is a private interface, so it will have less worse influence.

Reply:
Done